src/js/utils/FormatTime.js

```diff
@@ -0,0 +1,13 @@
+export default function (seconds) {
```

Comment:
For me it is unclear what FormatTime is. I'm assuming this module would eventually handle other time formats in the future.

Can this function have a name ? I think a good example to compare is Locale utils: https://github.com/grommet/grommet/blob/master/src/js/utils/Locale.js#L17

Also, we are trying to add tests as much as we can. This utils would be a good candidate for testing so that we can understand the formatting better.